frontend-ssr/components/cli_git/git_commands_cc.tsx

```diff
@@ -1,22 +1,66 @@
 import React from 'react';
```

Comment:
Let's rename the file. If the styles are called `GitCli.module.scss` then the component file should be called `GitCli.tsx`.